src/toil/test/wdl/builtinTest.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import os
+import subprocess
+import shutil
+import uuid
+
+from toil.version import exactPython
+from toil.test import ToilTest
+
+
+class WdlStandardLibraryWorkflowsTest(ToilTest):
```

Comment:
How big is the WDL standard library? Are we going to end up with 50 test functions in this class? Should we maybe (later?) switch to 50 test files instead? Or find a way to condense those 50 functions down as much as possible, so it's more like 50 asserts?